lib/category/category_intermediary_page.dart

```diff
@@ -52,20 +52,32 @@ class CategoryIntermediaryView extends StatelessWidget {
           case Status.success:
             final category = state.category!;
 
-            return ListView(
-              children: category.children
-                  .map(
-                    (category) => ListTile(
-                      title: Text(category.name),
-                      onTap: () {
-                        context.router.pushNamed(
-                          category.destination ??
-                              'category/${category.id}?title=${category.name}',
-                        );
-                      },
-                    ),
-                  )
-                  .toList(),
+            return SingleChildScrollView(
+              child: Column(
+                children: [
+                  // category header
+                  CategoryHeader(category: category),
+
+                  // subcategory list
+                  ListView(
```

Comment:
This can be a simple ListView no?

```dart
ListView(
  children: [
    CategoryHeader(category: category),
    ...category.children
      .map(
        (category) => ListTile(
          title: Text(category.name),
          onTap: () {
            context.router.pushNamed(
              category.destination ?? 'category/${category.id}?title=${category.name}',
            );
          },
        ),
      ).toList(),
  ]
)
```